### DISCUSSION TITLE
Add Yunohost package

### DISCUSSION THREAD

**Discussion by MagicLike** [JulianPrieber/littlelink-custom#217] [In progress] [answered]
### Discussed in https://github.com/JulianPrieber/littlelink-custom/discussions/178

<div type='discussions-op-text'>

<sup>Originally posted by **MagicLike** October  7, 2022</sup>
Idea: https://ruhr.social/@lx242/109128376624559478

Docs: https://yunohost.org/de/packaging_apps</div>

~~Please use [this](https://github.com/LittleLink-Custom/LittleLink-Custom_ynh) repo when creating the package.~~

**Comment by Tagadda** [comment, marked as answer]
:wave: https://github.com/YunoHost-Apps/linkstack_ynh :sparkles: 

You can eventually add this somewhere in the README / documentation :
```
[![Install LinkStack with YunoHost](https://install-app.yunohost.org/install-with-yunohost.svg)](https://install-app.yunohost.org/?app=linkstack)
```

**Reply by MagicLike** [reply]
Niice - thank you! I had already seen it, but forgot about this discussion here...
I will add it later to the docs and stuff.

If you want to update the screenshots and stuff a bit, take a look at https://github.com/LinkStackOrg/branding

As I have never created a yunohost package, how is it with access to the repo for the project maintainers? (Not that it is necessary :grin: )

**Reply by Tagadda** [reply]
> As I have never created a yunohost package, how is it with access to the repo for the project maintainers? (Not that it is necessary grin )

You've got an invitation :)
Checkout [the documentation](https://yunohost.org/en/packaging_apps_intro) and feel free to [join the chat room](https://yunohost.org/en/chat_rooms#applications-chat-room) if you have any question! :)
